### DISCUSSION TITLE
AGPL Licence

### DISCUSSION THREAD

**Discussion by brianteeman** [akeeba/panopticon#342] [General]
As the licence requires me to publish any changes or templates etc that I may make is there a required/preferred way of doing this? Do I for example have to notify you that I have published something

**Comment by nikosdion** [comment]
No, you don't have to explicitly notify me (the original author). 

The GNU AGPL 3+ license requires you to make the source code of your changes available to anyone who can access your service over a network. Since the login page does count as access, this is generally understood to mean that you need to have a link to download your modified source code, or a link to your forked repo, in a public location which is fairly obvious to all visitors to your installation. This could be the footer of Panopticon along with a link in the login page, or the footer of the public pages describing your service.

If you do make an interesting change which would benefit others I encourage you and would greatly appreciate it if you drop me a line, but it's by no means a legal requirement. It's just a nice gesture.
